readthedocs/builds/migrations/0007_add-automation-rules.py

```diff
@@ -66,7 +66,7 @@ class Migration(migrations.Migration):
                     "action_arg",
                     models.CharField(
                         blank=True,
-                        help_text="Value used for the action to perfom an operation",
+                        help_text="Value used for the action to perform an operation",
```

Comment:
This would require a Django migrations.

Reply:
ok, let me exclude all such lines with `help_text *=.*`

Reply:
even better -- while reviewing tuneups needed for pre-commit (e.g. #11201 and from there on) - ralized that it is `migrations` -- better be excludes altogether.

Reply:
sorry for the noise -- original is still in readthedocs/builds/models.py . I will add explicit skip for this phrase and TODO

Reply:
Note we can't skip the whole migrations directory since we are running safemigrations pre commit over it mainly, but also other checks as well